docker/server/Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends cmake git \
 	&& rm -rf /var/lib/apt/lists/*
 
 WORKDIR /opt
-RUN git clone https://github.com/microsoft/vcpkg --depth=1
+RUN git clone https://github.com/opentibiabr/vcpkg --depth=1
```

Comment:
Do we really need to keep our own repository?

Can't we just clone a specific tag to ensure it always works?

Reply:
I also don't think it's a good approach, but since it's the one that's currently being used, I just added a correction so the build won't break :D